python/pytorchserver/pytorchserver/model.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfserving
+import numpy as np
+import os
+from typing import List, Any
+import torch
+from torch.autograd import Variable
+import importlib
+import shutil
+import sys
+
+PYTORCH_FILE = "model.pt"
+
+class PyTorchModel(kfserving.KFModel):
+    def __init__(self, name: str, model_class_name: str, model_class_file: str, model_dir: str):
+        super().__init__(name)
+        self.name = name
+        self.model_class_name = model_class_name
+        self.model_class_file = model_class_file
+        self.model_dir = model_dir
+        self.ready = False
+
+    def load(self):
+        model_file = os.path.join(
+            kfserving.Storage.download(self.model_dir), PYTORCH_FILE)
+        model_class_file = os.path.join(
+            kfserving.Storage.download(self.model_dir), self.model_class_file)
+        model_class_name = self.model_class_name
+
+        # Load the python class into memory
+        sys.path.append(os.path.dirname(model_class_file))
```

Comment:
What do you think setting the path to `model_dir` here?

Reply:
but which file? We need to know the model_class_file to load into memory

Reply:
Ok, looking at an interface where we assume there is only one *.py file, and then we can simplify this.

Reply:
Done